tiatoolbox/models/abc.py

```diff
@@ -100,6 +100,7 @@ def preproc_func(self):
         """Return the current pre-processing function of this instance.
 
         The returned function is expected to behave as follows:
+        Example:
         >>> transformed_img = func(img)
```

Comment:
spacing is not consistent as we have used before. Please check https://github.com/TIA-Lab/tiatoolbox/blob/develop/tiatoolbox/wsicore/save_tiles.py